src/librustc/lint/builtin.rs

```diff
@@ -125,7 +126,11 @@ declare_lint! {
 declare_lint! {
     pub PRIVATE_IN_PUBLIC,
     Warn,
-    "detect private items in public interfaces not caught by the old implementation"
+    "detect private items in public interfaces not caught by the old implementation",
```

Comment:
we should probably update [the page on this in the forge](https://forge.rust-lang.org/compiler/bug-fix-procedure.html#issuing-future-compatibility-warnings) -- actually, that content should really move to rustc-guide I expect -- or at least some of it